atm.js

```diff
@@ -1,21 +1,107 @@
+/*
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+*/
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+window.onload = function(){
+  var checking = new Account(0);
+  var savings = new Account(0);
 
-}
 
-function checkingWithdrawal(amount){
+  var checkingDepositBtn = $("input[value='Deposit']").eq(0);
+  var checkingWithdrawBtn = $("input[value='Withdraw']").eq(0);
+  var checkingDisplay = $(".balance").eq(0);
+  var checkingInput = $("input[type='text']").eq(0);
 
-}
+  var savingsDepositBtn = $("input[value='Deposit']").eq(1);
+  var savingsWithdrawBtn = $("input[value='Withdraw']").eq(1);
+  var savingsDisplay = $(".balance").eq(1);
+  var savingsInput = $("input[type='text']").eq(1);
 
```

Comment:
how could we modify our constructor function definition or modify the instances created to attach these variables as properties on the appropriate object?